src/lib/device/virtual_dev.cpp

```diff
@@ -600,8 +648,12 @@ int VirtualDev::cp_progress_percent() { return 100; }
 ///////////////////////// VirtualDev Private Methods /////////////////////////////
 uint64_t VirtualDev::to_dev_offset(BlkId const& b, Chunk** chunk) const {
     *chunk = m_dmgr.get_chunk_mutable(b.chunk_num());
-    RELEASE_ASSERT(*chunk, "Chunk got null {}", b.chunk_num());
+    if (!(*chunk)) return INVALID_DEV_OFFSET;
     return uint64_cast(b.blk_num()) * block_size() + uint64_cast((*chunk)->start_offset());
 }
 
+bool VirtualDev::is_chunk_loaded(cshared< Chunk >& chunk) const {
```

Comment:
i wanted to set the parameter to only chunk_id. but if that, we need to get the chunk_id before 
```
if (sisl_unlikely(!is_chunk_available(chunk))) 
```
so , puttingthe action of getting chunk id inside `is_chunk_available` itself  seems more clean.
since it generates no impact to the logic, we can keep it like this and change it if necessary in the future.